server/services/discord/actions/on_reaction_add.js

```diff
@@ -9,4 +10,9 @@ const client = require('../init').client
  */
 client.on('messageReactionAdd', (reaction, user) => {
   console.log(reaction.emoji.name)
```

Comment:
log

Reply:
Not mine. It was already there. Should I delete it ?